examples/dev/Alpha_Config_UC

```diff
@@ -0,0 +1,39 @@
+## Use Case (UC): Running default config on the JUAMI Potentiostat for Alpha release.
+Intended User: Graduate student (GS) familiar with dev team and basic electrochemistry concepts. The instructions in this UC assume that the student has a resistor they can attach clips to.
+Assumes little to no knowledge of python. Assumes access to a Windows computer.
+
+### Before Installation: 
+1. Member of dev team gives GS a JUAMI Potentiostat with a 1kOhm resistor.
+1. GS installs python on their computer.
+
+### Installation:
+1. GS recieves email from member of dev team with link to release page.
+1. GS follow instructions on release page to install pytentiostat program.
+
+### Experimental Setup:
+1. GS plugs the USB cable into the computer and the potentiostat.
+1. GS attaches alligator clips to the resistor; one side with the working electrode and the other side with counter and reference electrodes.
+
+### Program Setup:
+1. GS copies the default example config file to an easy to navigate to directory (WD).
+1. GS opens up a console window.
```

Comment:
I might change "console window" to "command line/terminal" to sound more familiar.

Reply:
I agree